<ui version="4.0" >
 <class>Crystal</class>
 <widget class="QWidget" name="Crystal" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="icons/icons.qrc" >
    <normaloff>:/Crystal.png</normaloff>:/Crystal.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="ToolBar" native="1" name="toolBar" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dragStart" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text" >
        <string>Drag</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>SpaceGroup</string>
       </property>
       <property name="buddy" >
        <cstring>SpaceGroup</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SpaceGroup" />
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>1</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>A</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>latticeA</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="latticeA" >
         <property name="buttonSymbols" >
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals" >
          <number>5</number>
         </property>
         <property name="minimum" >
          <double>0.010000000000000</double>
         </property>
         <property name="maximum" >
          <double>45.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Alpha</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>latticeAlpha</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="latticeAlpha" >
         <property name="buttonSymbols" >
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals" >
          <number>5</number>
         </property>
         <property name="minimum" >
          <double>0.010000000000000</double>
         </property>
         <property name="maximum" >
          <double>179.900000000000006</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>B</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>latticeB</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="latticeB" >
         <property name="buttonSymbols" >
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals" >
          <number>5</number>
         </property>
         <property name="minimum" >
          <double>0.010000000000000</double>
         </property>
         <property name="maximum" >
          <double>45.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Beta</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>latticeBeta</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="latticeBeta" >
         <property name="buttonSymbols" >
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals" >
          <number>5</number>
         </property>
         <property name="minimum" >
          <double>0.010000000000000</double>
         </property>
         <property name="maximum" >
          <double>179.900000000000006</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>C</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>latticeC</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="latticeC" >
         <property name="buttonSymbols" >
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals" >
          <number>5</number>
         </property>
         <property name="minimum" >
          <double>0.010000000000000</double>
         </property>
         <property name="maximum" >
          <double>45.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Gamma</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>latticeGamma</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="latticeGamma" >
         <property name="buttonSymbols" >
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals" >
          <number>5</number>
         </property>
         <property name="minimum" >
          <double>0.010000000000000</double>
         </property>
         <property name="maximum" >
          <double>179.900000000000006</double>
         </property>
         <property name="singleStep" >
          <double>0.100000000000000</double>
         </property>
         <property name="value" >
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QLabel" name="label_9" >
     <property name="text" >
      <string>orientation matrix</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QTableWidget" name="orientationMatrix" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll" >
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="gridStyle" >
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="rowCount" >
      <number>3</number>
     </property>
     <property name="columnCount" >
      <number>3</number>
     </property>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="6" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Omega</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy" >
        <cstring>rotationOmega</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rotationOmega" >
       <property name="buttonSymbols" >
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="minimum" >
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.100000000000000</double>
       </property>
       <property name="value" >
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label_10" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Chi</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy" >
        <cstring>rotationChi</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rotationChi" >
       <property name="buttonSymbols" >
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="minimum" >
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.100000000000000</double>
       </property>
       <property name="value" >
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="2" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label_12" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Phi</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy" >
        <cstring>rotationPhi</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rotationPhi" >
       <property name="buttonSymbols" >
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals" >
        <number>3</number>
       </property>
       <property name="minimum" >
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>0.100000000000000</double>
       </property>
       <property name="value" >
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolBar</class>
   <extends>QWidget</extends>
   <header>Tools.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SpaceGroup</tabstop>
  <tabstop>latticeA</tabstop>
  <tabstop>latticeB</tabstop>
  <tabstop>latticeC</tabstop>
  <tabstop>latticeAlpha</tabstop>
  <tabstop>latticeBeta</tabstop>
  <tabstop>latticeGamma</tabstop>
  <tabstop>orientationMatrix</tabstop>
  <tabstop>rotationOmega</tabstop>
  <tabstop>rotationChi</tabstop>
  <tabstop>rotationPhi</tabstop>
 </tabstops>
 <resources>
  <include location="icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
